packages/credentials/tests/presentation-exchange.spec.ts

```diff
@@ -58,6 +65,24 @@ describe('PresentationExchange', () => {
       expect(actualSelectedVcJwts).to.deep.equal([btcCredentialJwt]);
     });
 
+    it('should return 0 verifiable credentials when they dont match the pd', async () => {
```

Comment:
minor nit but probably doesn't hurt to spell out:
```suggestion
    it('should return 0 verifiable credentials when they dont match the presentation definition', async () => {
```